MekHQ/src/mekhq/campaign/parts/MissingPart.java

```diff
@@ -369,12 +369,15 @@ public String checkScrappable() {
 
 	@Override
 	public String scrap() {
-		Part replace = findReplacement(false);
+        Part replace = findReplacement(false);
 		if(null != replace) {
 			replace.decrementQuantity();
-		}
+            return replace.getName() + " scrapped.";
+        }
+
 		skillMin = SkillType.EXP_GREEN;
-		return replace.getName() + " scrapped.";
+
+        return getName() + " scrapped.";
 	}
```

Comment:
Looks like this is tab spaced.

Reply:
I made no attempt to adjust whitespacing in files.